RPIPs/RPIP-60.md

```diff
@@ -41,29 +41,29 @@ This RPIP is part of a set of proposals motivated by a desire to rework Rocket P
 - If possible, the Veto Explanation Document SHOULD suggest alternative similar-but-non-damaging upgrades that could be considered
 - If this veto power is abused, the pDAO SHOULD discuss replacing the members of the security council
 
+## Rationale
+**Veto Explanation Document**  
+The explanation document is a mechanism intended to help the pDAO hold the security council to account for their decision-making around vetos. It gives more information as to the reasoning behind a veto decision and should help to inform pDAO decision-making post-veto. In an ideal world, the document accompanies the veto. The 24-hour grace period is intended to cover situations in which the security council must act within a limited time window.
+
+**Upgrade Delay Setting**  
+Due to the considerations around slowing down hotfixes [described above](#security-considerations), the initial upgrade delay setting is attempting to be the smallest that allows "enough" time for participants to exit the system ahead of a proposal if they deem that appropriate.
+
+Any upgrade will have `proposal.vote.delay.time` (currently 1 week) and `upgrade_delay` (initially set to 1 week). This isn't enough to match the RPL `unstaking_period` (currently 28 days). In normal circumstances, there will be more time available to exit. There is a leadup period to the sentiment poll (currently at least 1 week), there is a pDAO vote period (currently at least 2 weeks), there's a human action required to raise the oDAO proposal, and there are many human actions to place oDAO votes after `proposal.vote.delay.time`. These delays should provide enough breathing room to allow participants to exit ahead of a proposal despite the 28-day unstaking period.
```

Comment:
https://github.com/rocket-pool/RPIPs/pull/253#discussion_r1685926971

My suggestion for how to tweak for the new 1 week upgrade_delay

Reply:
Integrated that wording